app/poros/food.rb

```diff
@@ -0,0 +1,11 @@
+class Food
+  attr_reader :hits, :gtinUpc, :description, :brandOwner, :ingredients
+
+  def initialize(data, hits)
+    @hits = hits
```

Comment:
I don't mind that you're storing the count on every `Food` item, however an argument against this is that is repetitive to store the same piece of unchanging data each time and takes up space in memory. It would be something more concerning in a larger app or if we were handling very large amounts of data.